src/routing/Routes.tsx

```diff
@@ -128,7 +131,7 @@ export default class Routes extends React.Component<Props, State> {
             <Route exact key={3} path={locations.landSelectENS()} component={LandSelectENSPage} />,
             <Route exact key={4} path={locations.landAssignENS()} component={LandAssignENSPage} />,
             <Route exact key={5} path={locations.ensSelectLand()} component={ENSSelectLandPage} />
-            <Route exact path={locations.worlds()} component={WorldListPage} />,
+            <Route exact path={locations.worlds()} component={ToRenderWorldListPage} />,
```

Comment:
Be aware that if you access the site via URL, you might not be able to load the new component as the FF might not have been queried yet.